security-analysis/security-analysis-default/src/main/java/com/powsybl/security/impl/DefaultSecurityAnalysis.java

```diff
@@ -222,4 +237,30 @@ private void applyContingency(String workingVariantId, String postContVariantId,
         contingency.toTask().modify(network, computationManager);
     }
 
+    private void addMonitorInfos(Network network, StateMonitor monitor, Consumer<BranchResult> branchResultConsumer,
+                                 Consumer<BusResults> busResultsConsumer, Consumer<ThreeWindingsTransformerResult> threeWindingsTransformerResultConsumer) {
+        monitor.getBranchIds().forEach(branchId -> branchResultConsumer.accept(createBranchResult(network.getBranch(branchId))));
```

Comment:
I think it can throw an NPE if the branch does not exist : we should just ignore it instead